website/blog/2022-12-01-release-0.10-blog.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Podman Desktop Release v0.10.0
+description: Podman Desktop v0.10.0 has been released!
+slug: podman-desktop-release-0.10.0
+authors: [deekay2310]
+tags: [podman-desktop, release, kubernetes]
+hide_table_of_contents: false
+---
+
+Let's talk about all that is new with the latest release of Podman Desktop v0.10.0!
+
+<!--truncate-->
+
+## Improved Kubernetes Workflow
+Podman Desktop facilitates the application development process right from containerization to Kubernetes deployment. With a focus on improving the Kubernetes workflow, we’ve introduced a set of new features.
+
+**1. Custom Kubeconfig file path ([#780](https://github.com/containers/podman-desktop/pull/780))** 
+
+Starting v0.10.0, users can provide their kubeconfig path location instead of letting Podman Desktop use the default path i.e. `$HOME/.kube/config`. This can be done by going to `Settings > Preferences > Kubernetes: Kubeconfig`
+
+![custom kubeconfig file path](img/podman-desktop-release-0.10/custom-kubeconfig.png)
+
+**2. Play Kubernetes YAML from the Containers list ([#739](https://github.com/containers/podman-desktop/pull/739))**
+
+Development workflows can require developers to leverage existing Kubernetes YAML files. Podman Desktop now supports playing Pods or Containers from an existing Kubernetes YAML File. 
+
+This can be done in the ‘Containers’ screen of the application by clicking the `Play Kubernetes YAML` button and playing the existing YAML file.
+
+**3. Kube context automatically reloaded when updated ([#813](https://github.com/containers/podman-desktop/pull/813))**
+
+Previously, Podman Desktop captured the kube context only at startup. Any change to the kube context would not be detected after that. This required the user to restart Podman Desktop. With this change, one can now reload kube context without restarting Podman Desktop.
+
+Launch Podman Desktop and then try to change the current kube context either by using the tray icon or with `kubectl` tool and then try to use ‘Deploy to Kubernetes’ feature, the selected context should be displayed.
```

Comment:
they don't need to **try** this feature. The value of this feature, is that it is automatically handled for the developer and they don't have to worry about that :)

Reply:
https://github.com/containers/podman-desktop/pull/943/commits/ed51bbcba5b04ecdae8a6406aabbd2040153097a